api-reference/execution-details/get-execution-details.mdx

```diff
@@ -4,6 +4,16 @@ openapi: get /v1/execution-details
 
 <RequestExample>
 
+```javascript Node.js
+import fetch from 'node-fetch';
+
+const response = await fetch('https://api.novu.co/v1/execution-details', {
+  method: 'GET',
+  
+});
+const data = await response.json();
+```
```

Comment:
This is not the Node SDK method for this. Don't we have that?

Reply:
No, the link you sent me didn't have it. And so, as you said, I copied it from the old docs.